src/graphql-server/resolvers.js

```diff
@@ -3,13 +3,27 @@ import HyloDnaInterface from './dnaInterfaces/hyloDnaInterface'
 import HappStoreDnaInterface, { getHappDetails } from './dnaInterfaces/happStoreDnaInterface'
```

Comment:
the changes in this file were accidentally caught in this PR, you can ignore them they're not part of what I'm wanting you to look at.